apps/nest-server/README.md

```diff
@@ -1,9 +1,27 @@
-# nest-server
+# Nest Server
 
-This application was generated with [Nx](https://nx.dev).
+This application was generated using [Nx](https://nx.dev).
 
-## Running
+## Running the Application
 
-First run `npm run presto:up` to run a local Presto using `docker-compose` and then `npm run serve nest-server` to serve the application.
+To run this application, follow these steps:
 
-Make a `GET` request to `http://localhost:3000/api/call-centers` to see the response from querying the local Presto using `prestodb-js-client` library.
+1. Start a local Presto instance using `docker-compose` by running the following command:
+
+   ```bash
+   npm run presto:up
+   ```
+
+2. Once the Presto instance is up and running, you can start the Nest Server by running:
+
+   ```bash
+   npm run serve nest-server
+   ```
+
+3. After starting the Nest Server, you can make a `GET` request to the following URL to see the response from querying the local Presto container using the `prestodb-js-client` library:
+
+   ```
+   http://localhost:3000/api/call-centers
```

Comment:
do you want to have a curl command here or just say open a browser with this URL? That would be more specific. And maybe add a section with the sample outputs.

Reply:
Like the step 3 you have in the nextjs's README.

Reply:
A curl command sounds good 👌🏼 Will add sample outputs as well

Reply:
Done!